[BUG] Could not resolve "icons/valhalla_isochrone.svg"

### Select items which related to this issue

maplibre-gl-terradraw

### Bug Description

Hi,
Could you please suggest whether this error is expected on my project's build?

[ERROR] Could not resolve "icons/valhalla_isochrone.svg" [plugin angular-css-resource]
    node_modules/@watergis/maplibre-gl-terradraw/dist/maplibre-gl-terradraw.css:1:9143:
      1 │ ...tton{background:url(icons/valhalla_isochrone.svg);background-pos...

### Bug Reproduction

_No response_

### Bug Screenshots

_No response_

### Bug System Information

- OS: [e.g. Windows 10]
- Browser: [e.g. Google Chrome, Firefox]

```
Failed to compile

../../node_modules/@watergis/maplibre-gl-terradraw/dist/maplibre-gl-terradraw.css:4:0
Module not found: Can't resolve './icons/valhalla_isochrone.svg'

Import trace for requested module:
../../node_modules/@watergis/maplibre-gl-terradraw/dist/maplibre-gl-terradraw.css
./pages/examples/naksha-mapbox-draw.tsx

https://nextjs.org/docs/messages/module-not-found
```




while importing **@watergis/maplibre-gl-terradraw/dist/maplibre-gl-terradraw.cs**s i am having the error too

--- a/.changeset/afraid-areas-guess.md
+++ b/.changeset/afraid-areas-guess.md
@@ -1,0 +1,5 @@
+---
+'@watergis/maplibre-gl-terradraw': patch
+---
+
+fix: fixed bug of not embedding valhalla_isochrone.svg in the output css file by deleting maxSize setting in postcss.config.js

--- a/postcss.config.js
+++ b/postcss.config.js
@@ -6,8 +6,8 @@ export default {
 		postcssUrl({
 			url: 'inline',
 			filter: '**/*.svg',
-			optimizeSvgEncode: true,
-			maxSize: 14
+			optimizeSvgEncode: true
+			// maxSize: 14
 		}),
 		cssnano({
 			preset: 'default'

--- a/src/scss/maplibre-gl-terradraw.scss
+++ b/src/scss/maplibre-gl-terradraw.scss
@@ -380,7 +380,6 @@
 		}
 
 		th {
-			// background-color: #f8f9fa;
 			font-weight: 600;
 			font-size: 0.85rem;
 		}

